src/settings/handlers/LocalEchoWrapper.ts

```diff
@@ -40,31 +40,37 @@ export default class LocalEchoWrapper extends SettingsHandler {
     }
 
     public getValue(settingName: string, roomId: string): any {
-        const cacheRoomId = roomId ? roomId : "UNDEFINED"; // avoid weird keys
+        const cacheRoomId = roomId || "UNDEFINED"; // avoid weird keys
```

Comment:
we should probably be better in our typing:
```suggestion
        const cacheRoomId = roomId ?? "UNDEFINED"; // avoid weird keys
```